exporter/containerimage/writer.go

```diff
@@ -693,6 +718,11 @@ func patchImageConfig(dt []byte, descs []ocispecs.Descriptor, history []ocispecs
 
 	if epoch != nil {
 		for i, h := range history {
+			if srcImg != nil && i <= len(srcImg.History) {
```

Comment:
This needs to ensure that record is same (and probably also that matching diffID is the same).

Reply:
Fixed